example/e2e/detoxTest.spec.js

```diff
@@ -88,6 +88,36 @@ describe('Example', () => {
     }
   });
 
+  async function userOpensPickerTz({mode, display, interval, type}) {
+    await element(by.text(mode)).tap();
+    await element(by.text(display)).tap();
+    if (interval) {
+      await element(by.text(String(interval))).tap();
+    }
+    await element(by.id(type)).tap();
+  }
+
+  it('should update dateTimeText when date changes and TzLarge', async () => {
```

Comment:
Please expand the test description so that everyone who looks at it understands it right away (without asking themselves what is `TzLarge`?). You can use full sentences if you like, thank you!

Reply:
Fixed